### PR TITLE
Group all "worker" runs as "child" run

### LINKED ISSUE #16: child runs for starting jupyter lab and workers on headnode
Tom and Fred know what this means

### PR DESCRIPTION
**Pull request by fredms**
Group all "worker" runs as "child" run

### CODE REVIEW

File: dask_cloudprovider/providers/azure/azureml.py [hunk 1]
```diff
@@ -532,37 +532,48 @@ def print_links_ComputeVM(self):
...
-                , entry_script='start_worker.py' # pass scheduler ip from parent run
-                , environment_definition=self.environment_definition
-                , script_params=self.worker_params
-                , node_count=1
-                , distributed_training=MpiConfiguration()
-            )
+    def _scale_up(self, workers=1):
+        conda_dependencies = self.environment_definition.python.conda_dependencies
+        run_config = RunConfiguration(conda_dependencies=conda_dependencies)
+        run_config.target=self.compute_target
```
Comment (drabastomek) [inline dask_cloudprovider/providers/azure/azureml.py:627]: I think we need to add the docker_image spec here? Or are these child runs going to pick up the Environment? If not, for the GPU runs I will not have the right environment

Comment (drabastomek) [reply dask_cloudprovider/providers/azure/azureml.py:627]: We probably need to use something like this: 

```
          custom_docker_image='todrabas/mlads_rapids:fall19'
        , user_managed=True
```

Comment (drabastomek) [reply dask_cloudprovider/providers/azure/azureml.py:627]: This should probably default to whatever image we're using in the Environment

**Review by drabastomek — APPROVED** [review]

File: dask_cloudprovider/providers/azure/azureml.py (hunk 1 above)
Comment (fredms) [reply dask_cloudprovider/providers/azure/azureml.py:627]: yes.

**Pull request merged**